content/messaging/agent-chat-api/v3.2/rtm-reference/index.mdx

```diff
@@ -5304,6 +5304,26 @@ Informs about messages sent via the `multicast` method or by the system.
 | `content`   | Yes      |       |
 | `type`      | No       |       |
 
+### `chat_unfollowed`
+Informs that chat has been unfollowed. Useful in multiple connections scenarios, where one connection needs to know that another one unfollowed chat.
```

Comment:
```suggestion
Informs that a chat has been unfollowed. Useful in multiple connections scenarios, where one connection needs to know that another one unfollowed chat.
```

Reply:
Could we provide a bit more context in this description? Would it be correct to say: 

`Useful in multiple connections scenarios, where one app or integration needs to know that another one unfollowed chat.`